### ISSUE TITLE
MacOS group root is not available

### ISSUE THREAD

**Issue by noelmcloughlin** [saltstack-formulas/golang-formula#19]
MacOS is not supported.
```
          ID: golang|cache-archive
    Function: file.managed
        Name: /tmp/go1.10.linux-amd64.tar.gz
      Result: False
     Comment: Attempt 1: Returned a result of "False", with the following comment: "Group root is not available"
              Attempt 2: Returned a result of "False", with the following comment: "Group root is not available"
              Group root is not available
```

**Comment by noelmcloughlin** [comment]
Wrong archive is pulled down too.
```
          ID: golang|extract-archive
    Function: archive.extracted
        Name: /usr/local/golang/1.10
      Result: True
     Comment: /private/tmp/go1.10.linux-amd64.tar.gz extracted to /usr/local/golang/1.10/, due to absence of one or more files/dirs
     Started: 22:51:47.647573
    Duration: 10876.875 ms
```

**Comment by noelmcloughlin** [comment]
Fixed in #20
